src/ShapeCrawler/Drawing/IPicture.cs

```diff
@@ -17,6 +17,11 @@ public interface IPicture : IShape
     /// </summary>
     string? SvgContent { get; }
 
+    /// <summary>
+    ///     Gets or sets the transparency for this image. Range is 0.0 (fully opaque, default) to 1.0 (fully transparent).
+    /// </summary>
+    decimal Transparency { get; set; }
```

Comment:
How about the unit of measure be a percentage? I've been thinking about that since we were working on Crop.

![image](https://github.com/user-attachments/assets/2ccffaba-7c4c-437c-83b7-4301f7f27a2c)

Reply:
@ashahabov Sure. Do you mean use a `decimal` value with range 0-100? Or do you mean create a new type in `ShapeCrawler.Units` namespace, e.g. `Percentage`?

Reply:
Yes, I mean just use `decimal` with range 0-100.

Reply:
c240e25914cd671e47b839f3d62c3b6cf2afb077